packages/firestore/src/util/async_queue.ts

```diff
@@ -66,7 +70,13 @@ export const enum TimerId {
    * A timer used to retry transactions. Since there can be multiple concurrent
    * transactions, multiple of these may be in the queue at a given time.
    */
-  RetryTransaction = 'retry_transaction'
+  RetryTransaction = 'retry_transaction',
+
+  /**
+   * A timer used to retry operations scheduled via retryable AsyncQueue
+   * operations.
+   */
+  AsyncQueueRetry = 'async_queue_retry'
```

Comment:
nit: the inconsistency between "retry" at the beginning vs end of the enumeration entry is a little glaring. `RetryAsyncQueue` sounds a little weird. Maybe rename `RetryTransaction` to `TransactionRetry` to match?